core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/CommonLoadBalancer.scala

```diff
@@ -248,16 +274,30 @@ abstract class CommonLoadBalancer(config: WhiskConfig,
           // notice here that the activationPromises is not touched, because the expectation is that
           // the active ack is received as expected, and processing that message removed the promise
           // from the corresponding map
+          logging.info(this, s"received completion ack for '$aid', system error=${isSystemError}")(tid)
+
+          MetricEmitter.emitCounterMetric(
+            LoggingMarkers.LOADBALANCER_COMPLETION_ACK(controllerInstance, regularCompletionAck))
+
         } else {
           // the entry has timed out; if the active ack is still around, remove its entry also
           // and complete the promise with a failure if necessary
           activationPromises
             .remove(aid)
             .foreach(_.tryFailure(new Throwable("no completion or active ack received yet")))
+          val actionType = if (entry.isBlackbox) "blackbox" else "managed"
+          val blockingType = if (entry.isBlocking) "blocking" else "non-blocking"
+          val completionAckTimeout = calculateCompletionAckTimeout(entry.timeLimit)
+          logging.warn(
+            this,
+            s"forced completion ack for '$aid', action '${entry.fullyQualifiedEntityName}' (${actionType}), ${blockingType}, mem limit ${entry.memoryLimit.toMB} MB, time limit ${entry.timeLimit.toMillis} ms, completion ack timeout ${completionAckTimeout} from ${invoker}")(
```

Comment:
Minor Nit ... In general braces not needed for simple expression like in `${actionType}`

Reply:
I personally prefer to use the brace syntax (`${}`) even if it's not required. For me, it's more readable. If you think, I should remove, please let me know.

Reply:
Ack. For me Intellij just highlight thats _The enclosing block is redundant_ which turn the file marked as yellow instead of green! I would probably disable this inspection as this usage is in quite a few places

Reply:
I will go through my PR again and remove unnecessary braces (`${}`) to help you guys with IntelliJ.